happi/tests/test_device.py

```diff
@@ -56,32 +58,39 @@ class MyDevice(HappiItem):
                           (str, 'hat', 'hat'), (str, 5, '5'),
                           (bool, True, True), (bool, 0, False),
                           (bool, 'true', True), (bool, 'NO', False)])
-def test_type_enforce_ok(type_spec, value, expected):
+def test_type_enforce_ok(
+    type_spec: Tuple[type, Any, Any],
+    value: Tuple[type, Any, Any],
+    expected: Tuple[type, Any, Any]
+):
     entry = EntryInfo(enforce=type_spec)
     assert entry.enforce_value(value) == expected
 
 
 @pytest.mark.parametrize('type_spec, value',
                          [(int, 'cats'),
                           (bool, '24'), (bool, 'catastrophe')])
-def test_type_enforce_exceptions(type_spec, value):
+def test_type_enforce_exceptions(
+    type_spec: Tuple[type, Any, Any],
```

Comment:
I think this is just `type_spec: Type` and `value` is `Any`?

Reply:
Oh wow yes that's very wrong.